lisa/commands.py

```diff
@@ -37,7 +37,7 @@ def run(args: Namespace) -> int:
     run_timer = create_timer()
     run_error_message = ""
     try:
-        runner = RootRunner(runbook)
+        runner = RootRunner(runbook_builder=builder)
```

Comment:
This commit smells like it belongs in the past one together

Reply:
This change passes builder into runner for next change. Both the root runbook and builder exists in commands.py. So it can be changed.